Modules/ThirdParty/HDF5/src/itkhdf5/src/itk_hdf5_mangle.h

```diff
@@ -88,58 +88,28 @@ e.g. remove mangling for H5Fget_info but leave it for H5Fget_info1 and H5Fget_in
 #define H5AC_SOHM_LIST itk_H5AC_SOHM_LIST
 #define H5AC_SOHM_TABLE itk_H5AC_SOHM_TABLE
 #define H5AC_SUPERBLOCK itk_H5AC_SUPERBLOCK
-#define H5AC__close_trace_file itk_H5AC__close_trace_file
 #define H5AC__init_package itk_H5AC__init_package
-#define H5AC__open_trace_file itk_H5AC__open_trace_file
```

Comment:
Can you amend this commit to only add symbols? That should also re-trigger the CI.

Reply:
Not easily I don't think. As you know the file is generated by running the commands at the top of the file.

Why do you ask?  Do you think it has removed some that are in fact needed?

Reply:
It will remove many old symbols, but this will remove some platform-specific symbols for other platforms (e.g. Windows). The risk of somebody using the old HDF5 symbols for something else in their code is minimal.

Just keeping the additions can be done by manual diff viewer manipulation. Do you want to do it yourself @seanm, or do you want me to take over?

Reply:
Wouldn't it be better for someone to run the symbol mangling on Windows *also*, then merge the results?

Reply:
After all, there could be new Windows-only symbols in this new HDF that we'll miss otherwise.

Reply:
OK, I will do both (remove deletions and add Windows-specific symbols).

Reply:
The much bigger problem though is as per my first comment, figuring out why the shared lib build fails...

Reply:
@hjmjohnson I still think we are safer only adding symbols.

Reply:
@dzenanz  did you add the windows-specific symbols?

Reply:
If so, this could be resolved.

Reply:
It looks like I didn't. I will take a pass at it later this afternoon.

Reply:
Done. Perhaps the commits could be re-ordered to make Window additions immediately follow Unix update?